scripts/scil_compute_ssst_frf.py

```diff
@@ -0,0 +1,177 @@
+        epilog="References: [1] Tournier et al. NeuroImage 2007")
+
+    p.add_argument('input',
+                   help='Path of the input diffusion volume.')
+    p.add_argument('bvals',
+                   help='Path of the bvals file, in FSL format.')
+    p.add_argument('bvecs',
+                   help='Path of the bvecs file, in FSL format.')
+    p.add_argument('frf_file',
+                   help='Path to the output FRF file, in .txt format, '
+                        'saved by Numpy.')
+
+    p.add_argument(
+        '--mask',
+        help='Path to a binary mask. Only the data inside the mask will be '
+             'used for computations and reconstruction. Useful if no white '
+             'matter mask is available.')
+    p.add_argument(
+        '--mask_wm', metavar='',
+        help='Path to a binary white matter mask. Only the data inside this '
+             'mask and above the threshold defined by --fa will be used to '
+             'estimate the fiber response function.')
+    p.add_argument(
+        '--fa', dest='fa_thresh', default=0.7, type=float,
+        help='If supplied, use this threshold as the initial threshold '
+             'to select single fiber voxels. [%(default)s]')
+    p.add_argument(
+        '--min_fa', dest='min_fa_thresh', default=0.5, type=float,
+        help='If supplied, this is the minimal value that will be tried '
+             'when looking for single fiber voxels. [%(default)s]')
+    p.add_argument(
+        '--min_nvox', default=300, type=int,
+        help='Minimal number of voxels needing to be identified as single '
+             'fiber voxels in the automatic estimation. [%(default)s]')
+
+    p.add_argument(
+        '--roi_radius', default=10, type=int,
+        help='If supplied, use this radius to select single fibers from the '
+             'tensor to estimate the FRF. The roi will be a cube spanning '
+             'from the middle of the volume in each direction. [%(default)s]')
+    p.add_argument(
+        '--roi_center', metavar='tuple(3)',
+        help='If supplied, use this center to span the roi of size '
+             'roi_radius. [center of the 3D volume]')
+
+    add_overwrite_arg(p)
+
+    p.add_argument('--verbose', '-v', action='store_true',
+                   help='Produce verbose output.')
+
+    return p
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+
+    if args.verbose:
+        logging.basicConfig(level=logging.DEBUG)
+    else:
+        logging.basicConfig(level=logging.INFO)
+
+    assert_inputs_exist(parser, [args.input, args.bvals, args.bvecs])
+    assert_outputs_exists(parser, args, [args.frf_file])
+
+    vol = nib.load(args.input)
+    data = vol.get_data()
+
+    bvals, bvecs = read_bvals_bvecs(args.bvals, args.bvecs)
+
+    if not is_normalized_bvecs(bvecs):
+        logging.warning('Your b-vectors do not seem normalized...')
+        bvecs = normalize_bvecs(bvecs)
+
+    if bvals.min() != 0:
```

Comment:
We have the `scilpy.utils.bvec_bval_tool.check_b0_threshold` function to replace this.